python/lsst/ip/isr/ampOffset.py

```diff
@@ -235,16 +235,21 @@ def run(self, exposure):
             # ensuring that no values are erroneously added/subtracted.
             pedestals = np.nan_to_num(np.linalg.lstsq(A, B, rcond=None)[0])
 
-        metadata = exposure.getMetadata()
+        metadata = exposure.getMetadata()  # Exposure metadata.
+        self.metadata["AMPOFFSET_PEDESTALS"] = {}  # Task metadata.
```

Comment:
May I suggest a multiple binding here to be something like:
```ampOffsetPedestals = self.metadata["AMPOFFSET_PEDESTALS"] = {}
...
ampOffsetPedestals[ampName] = float(pedestal)

Reply:
Done!

Reply:
Ah, I ran some tests and found that multiple binding doesn't work with `TaskMetadata` the way we expect, i.e., this gives us empty results:
```python
        # Exposure metadata.
        metadata = exposure.getMetadata()
        # Task metadata.
        ampOffsetMetadata = self.metadata["AMPOFFSET_PEDESTALS"] = {}
        for amp, pedestal in zip(amps, pedestals):
            ampIm = exposure.image[amp.getBBox()].array
            ampIm -= pedestal
            ampName = amp.getName()
            # Add the amp pedestal to the exposure metadata.
            metadata.set(
                f"LSST ISR AMPOFFSET PEDESTAL {ampName}",
                float(pedestal),
                f"Pedestal level subtracted from amp {ampName}",
            )
            # Add the amp pedestal to the "Task" metadata as well.
            # Needed for Sasquatch/Chronograf!
            ampOffsetMetadata[ampName] = float(pedestal)
```
Debugging result:
```python
ipdb> ampOffsetMetadata
{'0': -0.20423207279935987, '1': -0.06727251761094916, '2': -0.28773895061785293, '3': 0.5592435410281624}
ipdb> self.metadata["AMPOFFSET_PEDESTALS"]
TaskMetadata(scalars={}, arrays={}, metadata={})
```

I'm reverting it.